src/test/java/it/pagopa/transactions/services/v1/TransactionServiceTests.java

```diff
@@ -2031,4 +2035,91 @@ void shouldRedirectToAuthorizationURIForValidRequestWithNPGWalletDetail() {
         assertEquals(calculateFeeResponseDto.getPaymentMethodName(), captureData.paymentMethodName());
     }
 
+    @Test
+    void shouldRedirectToAuthorizationURIForValidRequestWithNPGApmDetail() {
+        RequestAuthorizationRequestDto authorizationRequest = new RequestAuthorizationRequestDto()
+                .amount(100)
+                .paymentInstrumentId("paymentInstrumentId")
+                .language(RequestAuthorizationRequestDto.LanguageEnum.IT).fee(200)
+                .pspId("PSP_CODE")
+                .isAllCCP(false)
+                .details(
+                        new ApmAuthRequestDetailsDto()
+                );
+
+        Transaction transaction = TransactionTestUtils.transactionDocument(
+                it.pagopa.ecommerce.commons.generated.server.model.TransactionStatusDto.ACTIVATED,
+                ZonedDateTime.now()
+        );
+
+        /* preconditions */
+        CalculateFeeResponseDto calculateFeeResponseDto = new CalculateFeeResponseDto()
+                .belowThreshold(true)
+                .paymentMethodName("PaymentMethodName")
+                .paymentMethodDescription("PaymentMethodDescription")
+                .paymentMethodStatus(PaymentMethodStatusDto.ENABLED)
+                .bundles(
+                        List.of(
+                                new BundleDto()
+                                        .idPsp("PSP_CODE")
+                                        .taxPayerFee(200L)
+                        )
+                );
+
+        PaymentMethodResponseDto paymentMethod = new PaymentMethodResponseDto()
+                .name("paymentMethodName")
+                .description("desc")
+                .status(PaymentMethodStatusDto.ENABLED)
+                .id("paymentInstrumentId")
+                .paymentTypeCode("PO")
+                .addRangesItem(new RangeDto().min(0L).max(100L));
```

Comment:
We expect that payment method response should have `PaymentMethodManagementType` field valued with `ONBOARDABLE` here, right? If so add this value into mocked response

Reply:
Fixed with df64e91e5d99afc32223422ad98f6f19043e6140